packages/app/src/navigation/SidebarNavigationRow.vue

```diff
@@ -5,7 +5,6 @@
     :distance="8"
   >
     <div
-      v-bind="$attrs"
```

Comment:
With the update to `floating-vue`, the Tooltip binds the attributes of it's parent by default. This was causing two element have the `data-e2e-href` attribute. This was the only attribute being passed so we can safely remove the `v-bind`.